gradle/apps/service-builder/jndi/jndi-service/build.gradle

```diff
@@ -31,31 +25,4 @@ buildService {
 	apiDir = "../jndi-api/src/main/java"
 }
 
-group = "com.liferay.blade.samples.jndiservicebuilder"
-
-startTestableTomcat {
-	checkTimeout = 10 * 60 * 1000
-}
-
-testIntegration {
```

Comment:
Does this remove integration testing for this module?

Reply:
Yes, but testing still exists in the liferay-workspace version of the module